<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="mainTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="profileTab">
       <attribute name="title">
        <string>Profile</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="gameTab">
       <attribute name="title">
        <string>Game Board</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="TicTacToeBoard" name="XO_Board" native="true"/>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Scores:</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Player1:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="score1Label">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="player2Name">
                 <property name="text">
                  <string>Player2:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="score2Label">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="modLabel">
                 <property name="text">
                  <string>Mod:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="modText">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Reset Board</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>New Game</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Histoy</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="HistoryPage" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionSign_Out"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_us"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About qt</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSign_Out">
   <property name="text">
    <string>Sign Out</string>
   </property>
  </action>
  <action name="actionAbout_us">
   <property name="text">
    <string>About us</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TicTacToeBoard</class>
   <extends>QWidget</extends>
   <header location="global">tictactoeboard.h</header>
   <container>1</container>
   <slots>
    <signal>scoreUpdate()</signal>
    <slot>resetBoard()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>HistoryPage</class>
   <extends>QWidget</extends>
   <header location="global">historypage.h</header>
   <container>1</container>
   <slots>
    <slot>InitState()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>XO_Board</sender>
   <signal>scoreUpdate()</signal>
   <receiver>MainWindow</receiver>
   <slot>scoreUpdateLabels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSign_Out</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>signOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scoreUpdateLabels()</slot>
  <slot>signOut()</slot>
 </slots>
</ui>
